### ISSUE TITLE
hal results in invalid choice with 2.2.0

### ISSUE THREAD

**Issue by pzelnip** [rubik/radon#151]
I have Radon 2.2.0 installed, and doing a:

```
radon hal <nameofmyproject>
```
results in:
```
usage: radon [-h] [-v] {cc,raw,mi} ...
radon: error: invalid choice: 'hal' (choose from 'cc', 'raw', 'mi')
```
Yet the docs indicate that hal is a valid choice (https://radon.readthedocs.io/en/latest/commandline.html#the-hal-command).

**Comment by rwbogl** [comment]
I think that the docs were autogenerated for the most recent commit, which hasn't been pushed to PyPI yet. For instance, [the stable](https://radon.readthedocs.io/en/stable/index.html) version of the docs don't mention `radon hal` at all.

This could be fixed by changing the doc link to the stable version or pushing a new release to PyPI.

**Comment by rubik** [comment]
@rwbogl is right, the `hal` command was only present in the development version. I released version 2.3.0 to PyPI:
https://pypi.org/project/radon/
